BezierKit/Library/Path.swift

```diff
@@ -245,7 +245,17 @@ open class Path: NSObject, NSSecureCoding {
         return true
     }
 
-    #if !os(WASI)
+    #if os(WASI)
+    @available(*, deprecated, message: "unavailable on WASI due to missing plist support")
```

Comment:
These stub implementations are marked as `deprecated` instead of `unavailable` because unavailable declarations cannot satisfy protocol requirements.

Reply:
instead of marking the method as deprecated can we put the `NSSecureCoding` support in an extension of `Path` that is conditionally omitted for WASI?

Reply:
It makes sense 👍